ui/page/exchange/create_order_page.go

```diff
@@ -968,7 +968,7 @@ func (pg *CreateOrderPage) layoutDesktop(gtx C) D {
 									return txt.Layout(gtx)
 								}),
 								layout.Rigid(func(gtx C) D {
-									ticker := pg.AssetsManager.RateSource.GetTicker(fromCur + ext.MktSep + toCur)
+									ticker := pg.AssetsManager.RateSource.GetTicker(fromCur+ext.MktSep+toCur, false)
```

Comment:
Set to `true`?

Reply:
Oh crap. I agree should be true for all layout calls. Missed this.

Reply:
Resolved: https://github.com/crypto-power/cryptopower/compare/640545d487270723acf32e5043d4f093046630e6..86dc2534b54125c852a15abca5c73acb049951f8